test/config/config.go

```diff
@@ -32,4 +33,6 @@ func (c *CiliumTestConfigType) ParseFlags() {
 		"Provision Vagrant boxes and Cilium before running test")
 	flag.BoolVar(&c.HoldEnvironment, "cilium.holdEnvironment", false,
 		"On failure, hold the environment in its current state")
+	flag.StringVar(&c.SSHConfig, "cilium.SSHConfig", "",
+		"Command to get a different a different SSH-config")
```

Comment:
typo? "Command to get a different a different" should probably be "Command to get a different..."